fbw-a380x/src/systems/instruments/src/SD/Pages/Fctl/elements/Spoiler.tsx

```diff
@@ -20,15 +22,14 @@ export function deflectionToYOffset(deflection: number, maxDeflection: number):
     return normalizedDeflection * SCALE_HEIGHT;
 }
 
-export const Spoiler: FC<SpoilerProps> = ({ x, y, side, position }) => {
-    const deflectionInfoValid = true;
-    const spoilerDeflection = 0;
+export const Spoiler: FC<SpoilerProps> = ({ x, y, side, position, onGround }) => {
+    const [spoilerDeflection]: [number, (v: number) => void] = useSimVar(`L:A32NX_HYD_SPOILER_${position}_${side}_DEFLECTION`, 'number', 100);
     const maxDeflection = position >= 3 ? 50 : 30;
```

Comment:
```suggestion
    const maxDeflection = position >= 3 ? 50 : 35;
```
Max inner spoilers deflection should be 35°